src/data_access/FirebaseAccessObject.java

```diff
@@ -52,10 +51,52 @@ public void save(Project project) {
             ApiFuture<WriteResult> result = docRef.set(data);
     }
 
-    public Project getProject(String projectName) {
-        return projects.get(projectName);
+    public Project getProjectInfo(String projectName) {
```

Comment:
For getting information, almost all use cases will have some sort of get method.
Maybe we can use facade design principle for all get methods?

Reply:
Sure, I think this is possible.